### PR TITLE
Fix exception in pytorch classifier predict method

### LINKED ISSUE #1784: Exception in pytorch classifier predict method
**Describe the bug**
detach().cpu().numpy() is called twice in the following code (once before the if and once inside the if):

with torch.no_grad():
    model_outputs = self._model(torch.from_numpy(x_preprocessed[begin:end]).to(self._device))
output = model_outputs[-1]
output = output.detach().cpu().numpy().astype(np.float32)
if len(output.shape) == 1:
    output = np.expand_dims(output.detach().cpu().numpy(), axis=1).astype(np.float32)

This causes an exception. The internal call should be removed.

**System information (please complete the following information):**
 - OS: 12.4
 - Python version: 3.8
 - ART version or commit number: 1.11.0
 - TensorFlow / Keras / PyTorch / MXNet version

### PR DESCRIPTION
**Pull request by abigailgold**
Signed-off-by: abigailt <abigailt@il.ibm.com>

# Description

Remove redundant call to detach().cpu().numpy()

Fixes #1784

## Type of change

Please check all relevant options.

- [ ] Improvement (non-breaking)
- [X] Bug fix (non-breaking)
- [ ] New feature (non-breaking)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by beat-buesser — APPROVED** [review]
Hi @abigailgold Thank you very much!
